core/templates/dev/head/components/answer_group_editor.html

```diff
@@ -115,6 +116,7 @@
     </form>
 
     <div class="oppia-rule-save-cancel-buttons">
+      <p ng-show="editAnswerGroupForm.editDestForm.$invalid && editAnswerGroupForm.editDestForm.newStateName.$dirty" class="help-block oppia-form-error">Please use a non-empty description consisting of alphanumeric characters, spaces and/or hyphens.</p>
```

Comment:
This is in the right place, but can we make the font a bit smaller (say 0.85em)? Also I'd make the message more specific: "Please pick a card name consisting of ..."